server.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import sys
+import socket
+import struct
+
+def recv_all (sock, bufsize, flags=0):
+  result =b''
+  while (lenght_diff :=bufsize -len(result))>0:
+    result += sock.recv(lenght_diff, flags)
+  return result
```

Comment:
The indentation is weird.
4 spaces for each tab please.
In addition - whitespace before and after operators.